package.json

```diff
@@ -1,55 +1,52 @@
 {
-  "name": "next-js-boilerplate",
-  "version": "1.0.0",
-  "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "build-stats": "cross-env ANALYZE=true npm run build",
-    "export": "next export",
-    "build-prod": "run-s clean build export",
-    "clean": "rimraf .next out",
-    "lint": "next lint",
-    "lint-fix": "next lint --fix",
-    "build-types": "tsc --noEmit --pretty",
-    "prepare": "husky install"
-  },
-  "dependencies": {
-    "classnames": "^2.3.1",
-    "next": "^12.0.9",
-    "next-seo": "^4.29.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "styled-jsx-plugin-postcss": "^4.0.1"
-  },
-  "devDependencies": {
-    "@next/bundle-analyzer": "^12.0.9",
-    "@types/node": "^17.0.13",
-    "@types/react": "^17.0.38",
-    "@typescript-eslint/eslint-plugin": "^5.10.1",
-    "@typescript-eslint/parser": "^5.10.1",
-    "autoprefixer": "^10.4.2",
-    "cross-env": "^7.0.3",
-    "cssnano": "^5.0.16",
-    "eslint": "^8.7.0",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-airbnb-typescript": "^16.1.0",
-    "eslint-config-next": "^12.0.9",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-jsx-a11y": "^6.5.1",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.28.0",
-    "eslint-plugin-react-hooks": "^4.3.0",
-    "eslint-plugin-unused-imports": "^2.0.0",
-    "husky": "^7.0.4",
-    "lint-staged": "^12.3.2",
-    "npm-run-all": "^4.1.5",
-    "postcss": "^8.4.5",
-    "prettier": "^2.5.1",
-    "rimraf": "^3.0.2",
-    "tailwindcss": "^3.0.17",
-    "typescript": "^4.5.5"
-  },
-  "license": "ISC"
-}
+	"name": "next-js-boilerplate",
+	"version": "1.0.0",
+	"scripts": {
+		"dev": "next dev",
+		"build": "next build",
+		"start": "next start",
+		"build-stats": "cross-env ANALYZE=true npm run build",
+		"export": "next export",
+		"build-prod": "run-s clean build export",
+		"clean": "rimraf .next out",
+		"lint": "next lint",
+		"lint-fix": "next lint --fix",
+		"build-types": "tsc --noEmit --pretty"
+	},
+	"dependencies": {
+		"classnames": "^2.3.1",
+		"next": "^12.0.9",
+		"next-seo": "^4.29.0",
+		"react": "^17.0.2",
+		"react-dom": "^17.0.2",
+		"styled-jsx-plugin-postcss": "^4.0.1"
```

Comment:
We don't use this, i suppose we can remove this as well?

Reply:
removed